### DISCUSSION TITLE
给用户权限添加一个是否可下载的选择项可以吗？真的很有用

### DISCUSSION THREAD

**Discussion by im-dashan** [AlistGo/alist#5626] [Ideas]
建议给用户权限添加一个是否可下载的选择项可以吗？真的很有用
![image](https://github.com/alist-org/alist/assets/149475639/3f643035-a21b-42a0-90cd-0df738d1775e)

**Comment by im-dashan** [comment]
顶我上去吧

**Comment by luciferrrrrrrr** [comment]
同样需要这个功能，很实用

**Comment by pongfcnkl** [comment]
可以随便填代理url 就能不可下载了
